### ISSUE TITLE
Something wrong?

### ISSUE THREAD

**Issue by John1231983** [zhouyuangan/SE_DenseNet#1]
Hi, thanks for sharing your experiment results. I checked and found that you may have some redundant code in the _Dense layer that adds thr seblock in of convolution. You added it in loop (for) and after first convolution. Why do you add seblock in _Dense_layer again? Thanks

**Comment by zhouyuangan** [comment]
@John1231983 Hi, I found redundant code which you pointed, I think it's my mistake. For testing se_densenet, I have write some scripts to test se_densenet on cifar10 datasset today. And I will test whether redundant code removed can influence train-test result.Let's wait a few days to see result, and I will update README in a week.

**Comment by John1231983** [comment]
Good. And consider to remove seblock after transition block also. We often use seblock after denseblock only. Transition block only helps to reduce feature size

**Comment by zhouyuangan** [comment]
@John1231983 Yes, thank you for your suggestions, I will take it and make more comparative experiments.

**Comment by zhouyuangan** [comment]
@John1231983 Hi, John. I update my test result just now, pls check [it](https://github.com/zhouyuangan/SE_DenseNet/blob/master/README.md#conclusion). Thank you very much.

**Comment by John1231983** [comment]
Good job, But the result shows that with and without seblock has similar performance. In full code, you have added the sebock in Trans layer and Dense layer. How about add them in the loop function and remove in the _Transition and _Dense layer? I mean add in 
https://github.com/zhouyuangan/SE_DenseNet/blob/e5bb2fda0e49d50c274e29330c0948d3aa4a4376/se_densenet_full.py#L209

and 
https://github.com/zhouyuangan/SE_DenseNet/blob/e5bb2fda0e49d50c274e29330c0948d3aa4a4376/se_densenet_full.py#L219

And remove it in 
https://github.com/zhouyuangan/SE_DenseNet/blob/e5bb2fda0e49d50c274e29330c0948d3aa4a4376/se_densenet_full.py#L167

https://github.com/zhouyuangan/SE_DenseNet/blob/e5bb2fda0e49d50c274e29330c0948d3aa4a4376/se_densenet_full.py#L137

Thanks

**Comment by zhouyuangan** [comment]
@John1231983 It's worthy to make a test, I will update new result after job done, pls keep watching.Thanks.

**Comment by zhouyuangan** [comment]
The new [test result](https://github.com/zhouyuangan/SE_DenseNet#conclusion) has updated. 
I will [release train and test code](https://github.com/zhouyuangan/SE_DenseNet#todo) in  a few days.

**Comment by John1231983** [comment]
Good. That is what I expect. You also can try something as:

1. Sebock in loop only after denseblock ( remove seblock in transittion)
2. Seblock in loop only after transition block ( remove seblock in denseblock)

Actually, we do not know where seblock will be good for densenet architecture. In my opinion, the first case may get better result
